src/boot/measure.c

```diff
@@ -87,6 +90,21 @@ static int help(int argc, char *argv[], void *userdata) {
         return 0;
 }
 
+static char *normalize_phase(const char *s) {
```

Comment:
s is unused?

Reply:
ah, fun. I guess this only worked, since the only param we call this with is optarg anyway...